sequential.py

```diff
@@ -0,0 +1,32 @@
+import re
```

Comment:
Add docstring to explain what this file is about

Reply:
and for sure it shouldn't be located under root of project